keadm/app/cmd/util/cloudinstaller.go

```diff
@@ -78,11 +88,32 @@ func (cu *KubeCloudInstTool) InstallTools() error {
 	if err != nil {
 		return err
 	}
+
 	fmt.Println("Edgecontroller started")
 
 	return nil
 }
 
+//create device model and device CRDS
+func (cu *KubeCloudInstTool) createdevicemodel() error {
+	cmdf := fmt.Sprintf("kubectl create -f $GOPATH/src/github.com/kubeedge/kubeedge/build/crds/devices/devices_v1alpha1_devicemodel.yaml")
```

Comment:
Both kubectl commands will not work if KubeEdge code is not cloned already. keadm init will simply download binaries and not these crd file. Please fix this approach. 
Possibly you might have to download only these files in /etc/kubeedge/cloud/conf/ path and in kubectl command access this path to create crd files.

Reply:
Do you mean in createdevicemodel function we download devices_v1alpha1_devicemodel.yaml  file to /etc/kubeedge/cloud/conf  directory firstly and then  kubectl create?

Reply:
@samy2019